site/pages/react-native/getting-started.mdx

```diff
@@ -1,16 +1,3 @@
----
-title: Using within React Native applications
-description: A guide on integrating Account Kit within a React Native application
----
-
-# Using within React Native applications
-
-This hasn't been tested extensively, but it is possible to use Account Kit within a React Native application. We've built a simple example using expo [here](https://github.com/alchemyplatform/aa-sdk-rn-expo/tree/main). This guide assumes you're using Expo, but the same principles should apply to a bare React Native app as well.
-
-:::warning
-The Alchemy Signer is not yet supported in React Native applications. For 3rd party signer support in React Native, refer to your signer provider's documentation.
-:::
-
 ## Getting Started
 
 ### Upgrade to the latest Beta version of Expo
```

Comment:
Is this still required? How much of this page actually still needs to exist? It was primarily focused on getting started with expo

Reply:
Yeah I think it should, the rest of the guides go over specifictly the Signer, but for setting up the rest of `account-kit` having a guide would be quite useful IMO.